app/services/omdb_service.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+class OmdbService
+  include HTTParty
+  base_uri 'www.omdbapi.com'
+
+  def initialize(title, year)
+    @options = { query: { t: title, y: year, apikey: Rails.application.credentials.dig(:omdb, :apikey) } }
+  end
+
+  def search
+    self.class.get('', @options)
```

Comment:
`''` looks weird - how about creating CONST (for example: `ROOT_PATH = '/'`)?

Reply:
base_url is setup above, and in this ' ' you could specific additional urls is this example there's no additional path.